openrtb_ext/request_wrapper.go

```diff
@@ -755,6 +755,8 @@ type RegExt struct {
 	extDirty       bool
 	usPrivacy      string
 	usPrivacyDirty bool
+	GDPR           *int8
```

Comment:
Both new fields must be lower case since we don't want it exported.

Reply:
Makes sense. Corrected.